awx_collection/tests/integration/targets/tower_group/tasks/main.yml

```diff
@@ -1,6 +1,7 @@
 ---
-- name: Generate an inventory name
+- name: Generate names
   set_fact:
+    group_name: "group-{{ lookup('password', '/dev/null chars=ascii_letters length=16') }}"
```

Comment:
This might sound crazy, but my ideal name would be a _lot_ of contextual information like

> AWX-collection-test-tower_group-.............

Reply:
I'm with you, i'll update them